inbc-program/inbc/parser/parser.py

```diff
@@ -65,11 +65,11 @@ def parse_source_args(self) -> None:
 
         # Application Add Command
         app_add_parser = app_subparsers.add_parser('add')
-        app_add_parser.add_argument('--gpgKeyUri', '-gku', required=True,
+        app_add_parser.add_argument('--gpgKeyUri', '-gku', required=False,
```

Comment:
Consider adding a check to ensure that if `--gpgKeyUri` or `--gpgKeyName` is provided, the other is also provided, as they are both optional but should be used together if used at all. [important]